testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-azure/hive.properties

```diff
@@ -1,8 +1,9 @@
 connector.name=hive
 hive.metastore.uri=thrift://hadoop-master:9083
-hive.config.resources=/docker/presto-product-tests/conf/presto/etc/hive-default-fs-site.xml
-hive.azure.abfs-storage-account=${ENV:ABFS_ACCOUNT}
-hive.azure.abfs-access-key=${ENV:ABFS_ACCESS_KEY}
+fs.hadoop.enabled=true
```

Comment:
fs.hadoop.enabled=false

Reply:
Can this be caused by the `default` schema not having a `location` property?

Reply:
maybe try adding `tableLocation` to https://github.com/trinodb/trino/blob/ead5e9c6cb6e6c9e45767330f99cd5ad5babac4a/testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestAbfsSyncPartitionMetadata.java#L160

Reply:
so the abfs test was not testing abfs storage? good catch.